pkg/compute/init.go

```diff
@@ -259,10 +263,19 @@ func (c *InitCommand) Exec(in io.Reader, out io.Writer) (err error) {
 	}
 	defer os.RemoveAll(tempdir)
 
+	var ref plumbing.ReferenceName
+	if c.from == defaultTemplate {
+		ref = plumbing.NewBranchReferenceName(defaultTemplateBranch)
+	}
+	if c.branch != "" {
+		ref = plumbing.NewBranchReferenceName(c.branch)
+	}
```

Comment:
Does `ref` need a non-blank default?

Reply:
I tested this and read a bit of the go-git source, without you get the HEAD reference which was the same behaviour before this change.